macros.in

```diff
@@ -56,6 +56,7 @@
 %__hg			@__HG@
 %__bzr			@__BZR@
 %__quilt		@__QUILT@
+%__systemd_sysusers	%{_bindir}/systemd-sysusers
```

Comment:
This clearly needs a comment like "comment out to disable user and group creation"